### ISSUE TITLE
Workspace dependencies not installed when using pnpm

### ISSUE THREAD

**Issue by AndreasNasman** [ty-ras/meta#7]
When running `pnpm install`, a warning is shown about a needed `pnpm-workspace.yaml` file for the workspace dependencies to be installed.

```sh
❯ pnpm install
 WARN  The "workspaces" field in package.json is not supported by pnpm. Create a "pnpm-workspace.yaml" file instead.
```

As an effect, only the top level dependencies are installed.

**Comment by AndreasNasman** [comment]
Would be great to have full support for `pnpm` as it's quite attractive for new projects 😄

**Comment by stazz** [comment]
I agree. And I admit I didn't test with `pnpm` 🙈 Will take a look, either just include the file as per usual business and risk over-cluttering if user is actually using `yarn` or `npm`, or make some trick (pre-install script which generates the `pnpm-workspace.yaml` file?) to handle that. Maybe just include the file as part of the template, tricks tend to break :D

**Comment by stazz** [comment]
Based on what I finally formalated as my comment on #8 , the package manager should be asked as part of mandatory questions. It should have the following options:
- `yarn`,
- `npm`,
- `pnpm`, and
- `"Unspecified"`/`"Decide later"`

**Comment by stazz** [comment]
Fixed with version `0.13.2`.
